infra/stream/OutputStream.hpp

```diff
@@ -82,13 +82,53 @@ namespace infra
         StreamErrorPolicy& errorPolicy;
     };
 
+    template<class Parent, class TheWriter>
```

Comment:
Same comment for TheWriter, WriterType?

Reply:
Same answer, sounds good, I'll apply it :)